apps/dsb-client-gateway-api/src/app/modules/gateway/gateway.controller.ts

```diff
@@ -34,6 +34,16 @@ export class GatewayController {
   public async get(): Promise<GatewayResponseDto> {
     const health = await this.healthService.health();
 
+    const fqcn: string | undefined = this.configService.get<string>('AK_FQCN');
+    const topicName: string | undefined = this.configService.get<string>('AK_TOPIC_NAME');
+    const topicOwner: string | undefined = this.configService.get<string>('AK_TOPIC_OWNER');
+    const topicVersion: string | undefined = this.configService.get<string>('AK_TOPIC_VERSION');
+
+    let isAssociationKeyEnabled: boolean = true;
```

Comment:
This should be in constructor and it's value should be assigned there, ENV variables won't change without app restart